### PR TITLE
Responsive footer

### PR DESCRIPTION
**Pull request by OkelloSam21**
Fixes #

Added some code to the style.css file for a responsive footer.

Changes proposed 

-make the footer responsive

Test Methodology and Results

WhatsApp Image 2023-03-26 at 12 48 52 AM

### CODE REVIEW

**Comment by adityaraute** [comment]
HI Okello,

Is there any issue regarding this? 
If yes, could you please link it?
If not, could you please raise one?

**Comment by adityaraute** [comment]
I must say the footer looks great! Once you link an issue I'll merge the PR

File: index.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
```
Comment (adityaraute) [inline index.html:1]: Request you to not change this. Please revert it to what it was.

**Review by adityaraute — CHANGES_REQUESTED** [review]
Rest looks good. Only change the DOCTYPE tag.

**Comment by OkelloSam21** [comment]
> HI Okello,
> 
> Is there any issue regarding this? If yes, could you please link it? If not, could you please raise one?

Hello  @adityaraute 
I do not know How I can link this pull request to issue #38.  I need some help with that.

**Review by OkelloSam21 — COMMENTED** [review]
I have checked my files and I've resolved your requested changes.

**Comment by adityaraute** [comment]
This is the section for linking issues

**Review by adityaraute — APPROVED** [review]

**Pull request merged**